lang/tags/imperative.py

```diff
@@ -40,6 +40,9 @@ def code_state_go_to():
     def code_state_while():
         """Inserts while statement"""
 
+    def code_state_loop():
+        """Inserts loop statement"""
+
```

Comment:
This seems like the wrong place for this, I'm not sure I know many languages that have a bare "loop" construct.

Reply:
My thinking here was that this could be overloaded to represent infinite loops in general. For example, in C you might have something like `for(; ;)  `, or `while(true) ` in java, or `while True:` in python, or simply `for {}`.  Sometimes there are multiple ways to do an infinite loop in a language, and the community may prefer a particular method as the standard, which could be placed here. this could perhaps be changed to code state infinite loop?

Reply:
`code_state_infinite_loop` sounds like a reasonable idea to me.